src/actions/recipe/RecipeAction.test.js

```diff
@@ -0,0 +1,15 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { GET_RECIPE } from '../action-types';
+import { getRecipe } from './recipe';
+
+configure({ adapter: new Adapter() });
+
+describe('actions', () => {
+  it('should create an action to get recipes', () => {
```

Comment:
👍

**[comment]** Would be nice to do the setup for async action testing as well

Reply:
Okay. will do that